scripts/vote_3_step_1[packages][optional]build.sh

```diff
@@ -23,25 +23,25 @@ gpg_key="$(gpg --list-keys | grep -C1 "^pub" | tail -1 | sed -r 's|^\ +||')"
 #
 # Build packages
 #
-if [ -d packaging ]; then
-    rm -rf packaging
+if [ -d packages ]; then
```

Comment:
This whole block for building packages is obsolete, due to movement packages build to TC.

Reply:
which why I renamed it to Optional step, and I could stress it in the doc